src/test/java/org/folio/rest/impl/SynchronizationAPITests.java

```diff
@@ -160,6 +170,41 @@ public void declaredLostEventShouldBeCreatedAfterSynchronization() {
     checkSyncJobUpdatedByLoanEvent(syncJobId);
   }
 
+  @Test
+  public void agedToLostEventShouldBeDeletedAfterSynchronizationJobByUser() {
```

Comment:
They should be deleted BEFORE synchronization starts

Reply:
fixed